integrations/schemas/repository-configuration-schema.json

```diff
@@ -149,6 +149,9 @@
                   "concluded_license": {
                     "type": "string"
                   },
+                  "cpe": {
+                    "type": "string"
+                  },
```

Comment:
Out of interest: What's the order of properties in this file? It seems to neither be alphabetical, nor matching the order in Kotlin source code.

Reply:
I was asking myself the same question. The first properties are sorted alphabetically, so I tried to adhere to this convention; but then the order gets mixed up.